CardinalKit-Example/CardinalKit/Components/Tasks/Model/CoffeeChartDataSource.swift

```diff
@@ -41,14 +43,17 @@ extension CoffeeChartDataSource {
     static func fetchData(onCompletion: @escaping ([NSNumber: CGFloat]) -> Void) {
         var countPerAnswer = [NSNumber: CGFloat]()
         
-        CKSendHelper.getFromFirestore(collection: Constants.dataBucketSurveys, identifier: "SurveyTask-Coffee") { (document, error) in
-            
-            guard let payload = document?.data()?["results"] as? [[AnyHashable: Any]] else {
+        guard let authCollection = CKStudyUser.shared.authCollection else {
```

Comment:
Should we keep the auth collection inside of the library instead of the example app? 

Under the library model, the library would be in charge of determining where to save information (auth collection) and the developer can specify what the type of the information is (survey or `dataBucketSurveys`).

Reply:
I agree, auth collection should be inside the library.